File: Strings/caesar_cipher_encryptor.js
function caesarCipherEncryptor(string, key) {
	const alphabet = 'abcdefghijklmnopqrstuvwxyz';
	let result = '';

	for (let i = 0; i < string.length; i++) {
		let curChar = string[i];
		let curIdx = alphabet.indexOf(curChar);
		let newIdx = curIdx + key;
		result += alphabet[newIdx % alphabet.length];
	}
	return result;
}

// Time Complexity: O(n)
// Space Complexity: O(n)

File: Easy/caesar_cipher_encryptor.py
def caesarCipherEncryptor(string, key):
    encryptedCaesar = ''
    alphabet = list('abcdefghijklmnopqrstuvwxyz')

    for char in string:
        idx = alphabet.index(char)
        new_idx = (idx + key) % len(alphabet)
        encryptedCaesar += alphabet[new_idx]

    return encryptedCaesar

# Time Complexity: O(n)
# Space Complexity: O(n)
